views/keyword/_url_result_report.html

```diff
@@ -0,0 +1,26 @@
+<table class="table table-borderless">
+  <thead>
+    <tr>
+      <th>Search Results Urls</th>
+      <th>Top AdWord Advertisers Urls</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+        <ul class="table__list-group">
```

Comment:
It's kind of awkward to have lists inside table rows 😅 A bad option is to have nested tables 🤮 However a better approach would be NOT to use a table at all in this case. Just div + some CSS grid would be better.

Reply:
Used CSS grid